packages/next/next-server/server/next-server.ts

```diff
@@ -1,5 +1,6 @@
 import compression from 'next/dist/compiled/compression'
 import fs from 'fs'
+import chalk from 'chalk'
```

Comment:
Chalk is compiled and should be imported as such 

```suggestion
import chalk from 'next/dist/compiled/chalk'
```